src/components/organisms/about/about.scss

```
@@ -0,0 +1,29 @@
+@use "../../variables" as *;
+@use "../../molecules/message/message.scss" as *;
+.about {
+  @media screen and(min-width:$breakpoint-lg) {
+    max-width: 990px;
+    .img-group {
+      display: flex;
+      img {
+        width: 240px;
```

```suggestion
        flex: 1;
        min-width: 0;
```
親要素がflexなのでこうするとwidthを固定にしなくても等幅が実現できます。
min-widthを指定しているのは親要素をはみ出さないためで、以下の記事が詳しく説明しています。
https://qiita.com/mpyw/items/dfc63c1fed5dfc5eda26